ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/functions/[slug]/PauseButton.tsx

```diff
@@ -63,50 +79,82 @@ function PauseFunctionModal({
-        toast.success(`${functionName} was successfully paused`);
+        toast.success(`“${functionName}” was successfully paused`);
       }
     });
-    onClose();
+    onCloseWrapper();
   }
-
   function handleResume() {
     unpauseFunction({ fnID: functionID }).then((result) => {
       if (result.error) {
-        toast.error(`${functionName} could not be resumed: ${result.error.message}`);
+        toast.error(`“${functionName}” could not be resumed: ${result.error.message}`);
       } else {
-        toast.success(`${functionName} was successfully resumed`);
+        toast.success(`“${functionName}” was successfully resumed`);
       }
     });
-    onClose();
+    onCloseWrapper();
   }
 
   return (
     <AlertModal
       isOpen={isOpen}
-      onClose={onClose}
+      onClose={onCloseWrapper}
       onSubmit={isPaused ? handleResume : handlePause}
-      title={`Are you sure you want to ${isPaused ? 'resume' : 'pause'} this function?`}
+      title={`${isPaused ? 'Resume' : 'Pause'} “${functionName}”`}
       className="w-1/3"
     >
       {isPaused && (
-        <p className="p-6 pb-0">
-          This function will resume normal functionality and will be invoked as new events are
-          received. Events received during pause will not be automatically replayed.
-        </p>
+        <div>
+          <p className="p-6 pb-0">
+            This function will resume normal functionality and will be invoked as new events are
+            received. Events received during pause will not be automatically replayed.
+          </p>
+          <p className="p-6 pb-0">
+            <strong>
+              Are you sure you want to resume <code>{functionName}</code>?
+            </strong>
+          </p>
+        </div>
       )}
       {!isPaused && (
-        <ul className="list-inside list-disc p-6 pb-0 leading-8">
-          <li>No new runs will be queued or invoked.</li>
-          <li>Existing runs will continue to run to completion.</li>
-          <li>Events will continue to be received, but they will not trigger new runs.</li>
-          <li>Functions can be resumed at any time.</li>
-        </ul>
+        <div>
+          <ul className="list-inside list-disc p-6 pb-0 leading-8">
+            <li>No new runs will be queued or invoked.</li>
+            <li>Events will continue to be received, but they will not trigger new runs.</li>
+            <li>Functions can be resumed at any time.</li>
+            <li>
+              Function runs that are currently running will be:
+              <div style={{ marginTop: 4, marginLeft: 14 }}>
+                <SelectInput
+                  value={currentRunHandlingStrategy}
+                  options={currentRunHandlingOptions}
+                  onChange={setCurrentRunHandlingStrategy}
+                  placeholder="How should currently-running runs be handled?"
+                />
+              </div>
+            </li>
+          </ul>
+          <p className="p-6 pb-0">
```

Comment:
I think this should be at the top, before the list, as it's the primary focus of attention for users. Instead of making it <strong>, a className="font-semibold" should be enough!

Reply:
✅ Done in https://github.com/inngest/inngest/pull/1505/commits/f2e12d58cf78389acb3dd47f9d0c8a0d3827c064